### PR TITLE
Set dependencies version in repos file

### PR DESCRIPTION
**Pull request by rsanchez15**
Signed-off-by: RaulSanchez <raul@eprosima.com>

### CODE REVIEW

File: fastdds_monitor.repos [hunk 1]
```diff
@@ -2,28 +2,28 @@ repositories:
...
-        version: main
+        version: v0.4.0
     fastrtpsgen:
         type: git
         url: https://github.com/eProsima/Fast-DDS-Gen.git
-        version: master
+        version: v2.1.0
     fastrtpsgen/thirdparty/idl-parser:
         type: git
         url: https://github.com/eProsima/IDL-Parser.git
```
Comment (jparisu) [inline fastdds_monitor.repos:24]: Do we really need this in the Monitor?

**Comment by jparisu** [comment]
Why this do not require an approval?

File: fastdds_monitor.repos (hunk 1 above)
Comment (jparisu) [reply fastdds_monitor.repos:24]: Rebase over https://github.com/eProsima/Fast-DDS-monitor/pull/141 to remove them

**Comment by rsanchez15** [comment]
> Why this do not require an approval?

Because it does not go to `main` branch

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**